### ISSUE TITLE
Some dialog fields are not visually marked as required

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor4#3433]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

Some dialog fields are not visually marked as required, for example `Anchor Name` in Anchor dialog:

1. Go to any CKEditor 4 instance with link plugin.
2. Open Anchor dialog.

### Expected result

> For keyboard users there should be "*" mark that is required field for "Anchor Name" edit box.

### Actual result

> When keyboard users navigate "Anchor Name" edit box , there is no required field that is "*" mark .But while navigating using screen reader it is announcing it as a required field.

![image](https://user-images.githubusercontent.com/1061942/65142445-03882a80-da13-11e9-946e-66c9b961ba50.png)

## Other details

* Browser: All
* OS: All
* CKEditor version: `4.5+`
* Installed CKEditor plugins: -

**Comment by Comandeer** [comment]
Actually we have more such dialogs, e.g. in "Flash" one we also don't mark URL field as required. Probably it would be better to check all dialogs for such omissions.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/4869
